mllib/src/main/scala/org/apache/spark/mllib/util/MLUtils.scala

```diff
@@ -270,7 +270,7 @@ object MLUtils {
    */
   @Since("1.0.0")
   @Experimental
-  def kFold[T: ClassTag](rdd: RDD[T], numFolds: Int, seed: Int): Array[(RDD[T], RDD[T])] = {
+  def kFold[T: ClassTag](rdd: RDD[T], numFolds: Int, seed: Long): Array[(RDD[T], RDD[T])] = {
```

Comment:
BTW, fix the type of parameter ```seed``` of ```kFold``` from ```Int``` to ```Long```, because it was used by ```PartitionwiseSampledRDD``` which need ```seed``` with ```Long``` type.